vlasovsolver/cpu_moments.cpp

```diff
@@ -46,32 +46,35 @@ void calculateCellMoments(spatial_cell::SpatialCell* cell,
        skipMoments = true;
    }
 
-   // Clear old moments to zero value
-   if (skipMoments == false && computePopulationMomentsOnly == false) {
-      cell->parameters[CellParams::RHOM  ] = 0.0;
-      cell->parameters[CellParams::VX] = 0.0;
-      cell->parameters[CellParams::VY] = 0.0;
-      cell->parameters[CellParams::VZ] = 0.0;
-      cell->parameters[CellParams::RHOQ  ] = 0.0;
-      cell->parameters[CellParams::P_11] = 0.0;
-      cell->parameters[CellParams::P_22] = 0.0;
-      cell->parameters[CellParams::P_33] = 0.0;
-   }
+    // Clear old moments to zero value
+    if (skipMoments == false) {
```

Comment:
This line now changes behaviour if we have `computePopulationMomentsOnly == true`. Please fix.